pkg/apis/application/v1alpha1/types.go

```diff
@@ -108,6 +108,7 @@ type ApplicationStatus struct {
 	ComparisonResult  ComparisonResult     `json:"comparisonResult" protobuf:"bytes,1,opt,name=comparisonResult"`
 	RecentDeployments []DeploymentInfo     `json:"recentDeployments" protobuf:"bytes,2,opt,name=recentDeployment"`
 	Parameters        []ComponentParameter `json:"parameters,omitempty" protobuf:"bytes,3,opt,name=parameters"`
+	HealthState       HealthState          `json:"healthState,omitempty" protobuf:"bytes,4,opt,name=healthState"`
```

Comment:
Should this be HealthStatus instead of state? I tend to follow [this rule](https://stackoverflow.com/a/11229919):

status == how are you? [good/bad]
state == what are you doing? [resting/working]

Also, for the field name, lets shorten this to just "health". The type name can still be HealthStatus.

Then when accessing it will be app.Status.Health.Status.

Reply:
Good suggestion! done.